crates/runtime/src/capability.rs

```diff
@@ -56,6 +56,16 @@ mod wasmtime_bindings {
     });
 }
 
+#[allow(missing_docs)]
+mod old_logging_bindings {
```

Comment:
```suggestion
mod unversioned_logging_bindings {
```

supernit and might not even be necessary, but I think this would clearly communicate what they are and contributors won't look at this and have to wonder what old means